Runtime/Behaviours/CinemachineBrain.cs

```diff
@@ -244,15 +244,19 @@ private void OnGuiHandler()
             {
                 // Show the active camera and blend
                 var sb = CinemachineDebug.SBFromPool();
+#if CINEMACHINE_UNITY_IMGUI
```

Comment:
Need at the top:
#if !UNITY_2019_3_OR_NEWER
#define CINEMACHINE_UNITY_IMGUI
#endif